examples/media-services/multiple_storage_accounts/main.tf

```diff
@@ -0,0 +1,53 @@
+resource "azurerm_resource_group" "testrg" {
+  name     = "amstestrg"
+  location = "westus"
+}
+
+resource "azurerm_storage_account" "testsa" {
+  name                     = "amstestsa"
+  resource_group_name      = "${azurerm_resource_group.testrg.name}"
+  location                 = "${azurerm_resource_group.testrg.location}"
+  account_tier             = "Standard"
+  account_replication_type = "GRS"
+}
+
+#
+# Remove comments here and in the azure_media_services resource after the initial run
+# to see a storage account get added to an existing AMS account.
+#
+/*
+resource "azurerm_storage_account" "testsa2" {
+  name                     = "amstestsa2"
+  resource_group_name      = "${azurerm_resource_group.testrg.name}"
+  location                 = "${azurerm_resource_group.testrg.location}"
+  account_tier             = "Standard"
+  account_replication_type = "GRS"
+
+  tags {
+    environment = "staging"
+  }
+}
+*/
+
+resource "azurerm_media_services" "ams" {
+
+        name = "amstest"
+        location = "${azurerm_resource_group.testrg.location}"
```

Comment:
Could we align these? you can run `terraform fmt .` to automatically format all files in the directly

Reply:
I ran this for each of the examples. I had no idea this existed, but its a nice feature.